### PR TITLE
Create keyset when creating CSTG keypair

### PR DESCRIPTION
**Pull request by mcollins-ttd**
When a CSTG keypair is created for a site, we need to ensure the site also has a keyset.

### CODE REVIEW

File: src/main/java/com/uid2/admin/managers/KeysetManager.java [hunk 1]
```diff
@@ -108,6 +109,15 @@ public AdminKeyset createKeysetForClient(ClientKey client) throws Exception{
         return null;
     }
 
+    private Optional<AdminKeyset> getAdminKeyset(int siteId) {
```
Comment (jon8787) [inline src/main/java/com/uid2/admin/managers/KeysetManager.java:112]: could call it getAdminKeysetBySiteId() ? (since the primary key is keyset id )

File: src/test/java/com/uid2/admin/vertx/EncryptionKeyServiceKeysetsDisabledTest.java [hunk 2]
```diff
@@ -148,32 +145,6 @@ void addSiteKey() throws Exception {
     }
```
Comment (jon8787) [inline src/test/java/com/uid2/admin/vertx/EncryptionKeyServiceKeysetsDisabledTest.java:146]: why are these deleted?

File: src/test/java/com/uid2/admin/vertx/ClientSideKeypairServiceTest.java [hunk 3]
```diff
@@ -257,6 +257,13 @@ void addKeypair(Vertx vertx, VertxTestContext testContext) throws Exception {
 
         post(vertx, "api/client_side_keypairs/add", jo.encode(), ar -> {
             assertTrue(ar.succeeded());
+            final ArgumentCaptor<ClientSideKeypair> keypair = ArgumentCaptor.forClass(ClientSideKeypair.class);
+            try {
+                verify(this.keysetManager).createKeysetForClientSideKeypair(keypair.capture());
```
Comment (jon8787) [inline src/test/java/com/uid2/admin/vertx/ClientSideKeypairServiceTest.java:262]: does anything test createKeysetForClientSideKeypair itself?

**Review by jon8787 — APPROVED** [review]

File: src/test/java/com/uid2/admin/vertx/ClientSideKeypairServiceTest.java (hunk 3 above)
Comment (mcollins-ttd) [reply src/test/java/com/uid2/admin/vertx/ClientSideKeypairServiceTest.java]: Added tests

File: src/test/java/com/uid2/admin/vertx/EncryptionKeyServiceKeysetsDisabledTest.java (hunk 2 above)
Comment (mcollins-ttd) [reply src/test/java/com/uid2/admin/vertx/EncryptionKeyServiceKeysetsDisabledTest.java:146]: The method they were testing has stopped being used a while back.
